tests/base.py

```diff
@@ -29,10 +29,10 @@
 from yellowbrick.exceptions import ImageComparisonFailure
 
 
-def is_winconda_env():
-    return (os.name == 'nt' and
-            os.path.exists(os.path.join(sys.prefix, 'conda-meta')))
-
+def linux_pypi():
+    linux_os = sys.platform.startswith('linux')
+    pypi = not os.path.exists(os.path.join(sys.prefix, 'conda-meta'))
+    return linux_os and pypi
```

Comment:
see my comment about naming conventions below and how this might transform a bit.

Also I think it is appropriate to call this function and set the constant in the `Module Constants` below

Reply:
I see what you mean here - will make that change accordingly so the constant is imported instead of the function.

Reply:
Implemented in https://github.com/DistrictDataLabs/yellowbrick/pull/883/commits/2e22c3e3bb583f175941dff8721213fcca652f38

Reply:
https://github.com/DistrictDataLabs/yellowbrick/blob/07f253e8c4d42639f9e657f7c3e9fbc11dd1a83e/tests/base.py#L32-L35

Reply:
https://github.com/DistrictDataLabs/yellowbrick/blob/2e22c3e3bb583f175941dff8721213fcca652f38/tests/test_base.py#L29

Reply:
this is looking great!